pkg/resources/sop_url.go

```diff
@@ -21,6 +21,7 @@ const (
 	SopUrlAlertsAndTroubleshooting                            = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md"
 	sopUrlCloudResourceDeletionStatusFailed                   = "https://github.com/RHCloudServices/integreatly-help/tree/master/sops/2.x/alerts/clean_up_cloud_resources_failed_teardown.asciidoc"
 	sopUrlSendGridSmtpSecretExists                            = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/sendgrid_smtp_secret_not_present.asciidoc"
+	sopUrlDeadMansSnitchSecretExists                          = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/sendgrid_deadmanssnitch_secret_not_present.asciidoc"
```

Comment:
URL for sop does not exist

Reply:
Thank you very much Jim. Sorry, forgot. I will add it asap.

Reply:
Done. SOP created. - https://github.com/RHCloudServices/integreatly-help/pull/812